### PR TITLE
Fix PHPDoc and Co.

### PR DESCRIPTION
**Pull request by LukasReschke**
Ref https://github.com/owncloud/core/pull/14929#discussion_r26501240

cc @nickvergessen @PVince81

### CODE REVIEW

File: apps/files_sharing/lib/external/scanner.php [hunk 1]
```diff
@@ -31,6 +30,8 @@ public function scan($path, $recursive = self::SCAN_RECURSIVE, $reuse = -1) {
 	 *
 	 * @param string $file file to scan
 	 * @param int $reuseExisting
+	 * @param int $parentId
+	 * @param array | null $cacheData existing data in the cache for the file to be scanned
```
Comment (nickvergessen) [inline apps/files_sharing/lib/external/scanner.php]: `{@inheritDoc}` everywhere?

Comment (LukasReschke) [inline apps/files_sharing/lib/external/scanner.php]: Has another PHPDoc as the parent :see_no_evil: 

> 	 * If an exception happened while accessing the external storage,
	 * the storage will be checked for availability and removed
	 * if it is not available any more.

**Comment by nickvergessen** [comment]
Looks good :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**